api/queries_repo.go

```diff
@@ -524,6 +525,35 @@ func ForkRepo(client *Client, repo ghrepo.Interface, org string) (*Repository, e
 	}, nil
 }
 
+func LastCommit(client *Client, repo ghrepo.Interface) (*git.Commit, error) {
+	var responseData struct {
+		Repository struct {
+			DefaultBranchRef struct {
+				Target struct {
+					Commit struct {
+						History struct {
```

Comment:
Instead of reading `History` off of a Commit object, you can read its `oid` field which will be the commit SHA.

```graphql
query ($repoOwner: String!, $repoName: String!) {
	repository(owner: $repoOwner, name: $repoName) {
		defaultBranchRef {
			name
			target {
				... on Commit {
					oid
					messageHeadline
				}
			}
		}
	}
}
```